Shared/src/com/pacoapp/paco/shared/util/SearchUtil.java

```diff
@@ -0,0 +1,237 @@
+package com.pacoapp.paco.shared.util;
```

Comment:
For the jsqlparser jar, and any other jars that will be used between the Android sql code and the server sql code, please move the jars into shared/lib. They will then get copied into the Android dir (Paco) and the server dir (Paco-Server) by the build system.